lxd/api_project.go

```diff
@@ -319,6 +323,28 @@ func projectsPost(d *Daemon, r *http.Request) response.Response {
 		return response.BadRequest(err)
 	}
 
+	// Create the new per-project daemon configs.
```

Comment:
Could you expand the comment here to explain that this is needed in order to allow the potential for setting these per-member config keys later. Is that right?